src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const FORMAT_SOURCE = sourceString.split('\n').join('');
+  const STYLE_ARRAY = FORMAT_SOURCE.split(';');
+
+  const STYLE_OBJECT = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `STYLE_OBJECT`. It will decrease code readability. Consider using more descriptive names, such as `styleObject` or `cssObject`.